semcore/feature-popover/src/FeaturePopover.jsx

```diff
@@ -132,15 +129,15 @@ class FeaturePopoverPopper extends Component {
     } = this.asProps;
 
     return sstyled(styles)(
-      <Popper.Popper disableEnforceFocus zIndex={zIndex}>
+      <Popper.Popper disableEnforceFocus zIndex={zIndex} role={'dialog'} tabIndex={0} autoFocus>
```

Comment:
is this the right node for role=dialog and so on? 
I should be able to define an aria-label for the popper, but when I do this:
`<FeaturePopover.Popper aria-label='New feature: Export'`
aria-label goes to another node, not the dialog
![image](https://github.com/user-attachments/assets/952d5582-152c-4cdd-8e7e-07c103e98d76)
or should I set the aria-label to another element?

Reply:
No, I did the role incorrectly, I corrected it